docs/kfs_example.md

```diff
@@ -0,0 +1,137 @@
+# Preview support of KServe gRPC API{#ovms_kserve}
+
+OpenVINO Model Server 2022.2 release introduce preview of support for KServe gRPC API.
+
+Inference supports putting tensor buffers either in `ModelInferRequest`'s [`InferTensorContents`](https://github.com/kserve/kserve/blob/master/docs/predict-api/v2/grpc_predict_v2.proto#L155) and [`raw_input_contents`](https://github.com/kserve/kserve/blob/master/docs/predict-api/v2/grpc_predict_v2.proto#L202). There is no support for BF16 data type and there is no support for using FP16 in `InferTensorContents`.
+In case of sending raw images jpeg files BYTES data type should be used and data should be put in `InferTensorContents`'s `bytes_contents`.
+
+In current release there is support for:
+* Model Metadata
+* Inference
+
+In current release there is no support for:
+* Server Live
+* Server Ready
+* Server Metadata
+* Model Ready
```

Comment:
Model Ready will be supported in this release, right?

Reply:
Will be done in separate MR